tools/purge_from_cache.cmd

```diff
@@ -0,0 +1,162 @@
+:: MIT License
+::
+:: Copyright (c) 2021 Eric Derewonko
+::
+:: Permission is hereby granted, free of charge, to any person obtaining a copy
+:: of this software and associated documentation files (the "Software"), to deal
+:: in the Software without restriction, including without limitation the rights
+:: to use, copy, modify, merge, publish, distribute, sublicense, and\or sell
+:: copies of the Software, and to permit persons to whom the Software is
+:: furnished to do so, subject to the following conditions:
+::
+:: The above copyright notice and this permission notice shall be included in all
+:: copies or substantial portions of the Software.
+::
+:: THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+:: IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+:: FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+:: AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+:: LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+:: OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+:: SOFTWARE.
+::
+:: Purpose:
+:: This batch will iterate over cached thumbnail files and check if the image number is still in the library.db
+
+
+@echo OFF
+pushd %~dp0
+
+:defaults
+:: default values
+set cache_dir=
+set dryrun=1
+set mipmap_ext=jpg
+set action=echo found stale mipmap: 
+call %~dp0\common.cmd
```

Comment:
In the original script, purge_from_cache.sh, the variables are set in this script.  tools/common.sh detects GNU readlink or realpath for resolving symbolic links.

Reply:
I figured that much, but what are the steps you used to make sure it worked?